cmd/log-generator/log-generator.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"math"
+	"math/rand"
+	"os"
+	"path"
+	"runtime"
+	"strconv"
+	"sync"
+	"time"
+)
+
+const (
+	layoutFormat    = "02 Jan 06 15:04:05 MST"
```

Comment:
I don't think we need to specify this format ourselves. Isn't this one of the ISO or RFC standards?

Reply:
Not sure, just copy/pasted this code.

Reply:
I do not see an exact match in this table:
https://www.geeksforgeeks.org/time-formatting-in-golang/#
The closest is
```
RFC822
“02 Jan 06 15:04 MST”
```
But it doesn't have seconds.